Root/MT2.cxx

```diff
@@ -0,0 +1,161 @@
+#include "SusyNtuple/MT2.h"
```

Comment:
The original code consists of only header files. I split the two files (MT2.h and MT2_ROOT.h) into source files (with the implementations) and header files (with the definitions). Mostly because I didn't like only header files :)